tests/Api/CachedIdentifiersExtractorTest.php

```diff
@@ -45,7 +45,9 @@ public function testFirstPass()
 
         $identifiersExtractor = new CachedIdentifiersExtractor($cacheItemPool->reveal(), $decoration->reveal(), null);
 
-        $this->assertEquals(['id' => 1], $identifiersExtractor->getIdentifiersFromItem($dummy));
+        $expectedResult = ['id' => 1];
+        $this->assertEquals($expectedResult, $identifiersExtractor->getIdentifiersFromItem($dummy));
+        $this->assertEquals($expectedResult, $identifiersExtractor->getIdentifiersFromItem($dummy), 'Trigger the local cache');
```

Comment:
Not really for the coverage. You're right @teohhanhui, it doesn't verify that the local cache actually works (it's very hard to test reliably), however it tests if the local cache don't introduce a bug.

For instance, if some one contribute this patch:

```patch
-return $this->memoryCache[$resourceClass];
+return $this->memoryCache[$resourceC];
```

The test will detect the problem. Without this line, it will not.